v2/.goreleaser.yaml

```diff
@@ -0,0 +1,145 @@
+before:
+  hooks:
+    ## this will generate the sources and an initial binary, which will be overridden by the `builds` part
+    - make -e VERSION="{{.Version}}" build
+
+dist: ./_build/dist
+
+builds:
+  ## we generate two binaries for the agent:
+  ## 1. regular, production optimized
+  ## 2. debug, with no optimization or inlining
+  - &agent-build
+    id: jaeger-agent
+    dir: ./_build/agent
+    binary: jaeger-agent
+    flags: 
+      - -trimpath
+    gcflags:
+      - all=-N -l
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
```

Comment:
Do we need to support windows in v2 as well as we do in v1 ?

Reply:
Yes. I temporarily removed as I hit an error and forgot to put it back. Thanks for spotting it!